modules/luigi/phrases/greeting.yml

```diff
@@ -0,0 +1,16 @@
+type: qa                                    # remark types: question answering (qa), phrases
+name: "greet_general"                              # intent or topic
+Q:                                          # possible questions
+- "Guess what! i'm selling ice cream. want some?"
+- "Would you like to have an ice cream?"
+- "YO YO YO! IT IS ICE CREAM TIME! DO YOU WANT SOME ICE CREAM?"
+- "You look like you could use some ice cream! Do you want some?"
+A:                                          # answers
+  SUCCESS:
+  - "I knew it - everyone loves ice cream! well, what can I get you? I have chocolate and vanilla"
```

Comment:
I think you might have to remove the `I have chocolate and vanilla` since we are not yet sure what flavors we will have in the end.

Reply:
Also, I have added a state that will explain what flavors we have so there is no need to already say that here :)

Reply:
I know, this was before your push, I'll adjust for all flavors in the future. :)